ci/test.sh

```diff
@@ -126,11 +126,11 @@ fi
 CARGO_ARGS=`get_cargo_args`
 
 # Ensure we build without warnings
-cargo rustc --lib $CARGO_ARGS -- -Dwarnings
+(cd bindgen-tests && cargo rustc --lib $CARGO_ARGS -- -Dwarnings)
```

Comment:
This was testing that bindgen built without warnings, but is no longer doing that is it?